### ISSUE TITLE
[Metrics] Create pull request from Desktop

### ISSUE THREAD

**Issue by billygriffin** [desktop/desktop#6772]
We would like to understand how frequently people are creating pull requests today using GitHub Desktop in order to establish a baseline prior to incorporating "Create PR" as one of the top-level suggested next steps in the "no local changes" view. Note: This is a subset of #6714 

I'd like this to be on production for at least two weeks prior to the "create PR" suggested next step shipping, so ideally this would go into next week's release (1.6.2).

Question - Will we need to instrument it separately with the menu item, keyboard shortcut, and suggested next step? Or do we care the pathway folks take and just want to add one overall metric?

Basically, I'd love for us to be able to see whether incorporating it as a prominent suggestion materially increases the % of people using the PR workflow via Desktop.

cc: @niik and anyone else who wants to weigh in

**Comment by billygriffin** [comment]
@outofambit Thanks! I’m not sure I agree, so I’m going to think out loud for a moment to try to unpack why. If we’re only instrumenting the blank slate action, then there’s no need for this to ship before that action ships to proud. I think our primary goal here is to understand people using the PR creation action, and whether adding it as a suggested next step materially increases the overall usage via Desktop. If we’re solely instrumenting the blank slate button, we won’t get a sense of baseline or whether it increases that core GitHub workflow.

I’m not sure they need to be tracked separately, but possibly just any PR creation action from Desktop be tracked as a metric. Then, as long as we get that shipped to prod for some period prior to the “create PR” suggested next step shipping to prod, we’ll get a good feel for how much it’s influencing people’s use of the PR workflow from Desktop, if at all. Thoughts?

**Comment by outofambit** [comment]
@billygriffin that's fair. I think my use of "MVP" was an overstatement. I agree the real MVP of this metric is just raw "number of PRs created through desktop" no matter the source. we could always instrument further later to get source breakdown if we get curious. :)
